app/views/bookmarks/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1>Your Bookmarks</h1>
```

Comment:
WIll have to be adapted to the new cards. 
I could put my cards in a partial ?

Reply:
yes, left <%= render for your partial once your branch is merged